### ISSUE TITLE
Setting enemy spawns to zero

### ISSUE THREAD

**Issue by Camden-Tech** [BananaPuncher714/LethalQuantities#6]
For some reason it doesnt prevent the entities from spawning.  I have a few other mods but none of them should prevent this behavior.

**Comment by Camden-Tech** [comment]
I did do this in multiplayer so Ill check to see if it works with no other players on

**Comment by Camden-Tech** [comment]
nope no difference

**Comment by BananaPuncher714** [comment]
I will do some testing

**Comment by BananaPuncher714** [comment]
How are you setting the enemy spawns to 0? Are you changing the rarity, changing the spawning amount, changing the enemy spawning curves, limiting the power count, or changing the max count? Also, have you enabled the config?

**Comment by Camden-Tech** [comment]
I am just changing the enemy spawn amount. I did enable the config, and that seems to do something (I saw the test bunker spider).

Here are my configs:
[Enemies.txt](https://github.com/BananaPuncher714/LethalQuantities/files/13720237/Enemies.txt)
[DaytimeEnemies.txt](https://github.com/BananaPuncher714/LethalQuantities/files/13720238/DaytimeEnemies.txt)
[OutsideEnemies.txt](https://github.com/BananaPuncher714/LethalQuantities/files/13720239/OutsideEnemies.txt)

**Comment by BananaPuncher714** [comment]
I did some testing, and it seems that despite setting the max enemy spawn count to 0, there is a very small chance that it does not apply. I haven't quite been able to reproduce it consistently, but just to make sure it doesn't happen again, I added in a check to ensure that the max enemy count must be greater than 0 to be added to the list of spawnable enemies. For future reference, it is better to set the rarity to 0 if you want to disable enemy spawns.
